src/js/components/PostCard/PostCard.js

```diff
@@ -142,7 +144,12 @@ class PostCard extends React.Component<Props, State> {
             ))}
           </div>
         )}
-        <div className="d-flex flex-column flex-md-row justify-content-between p-2 border-bottom">
+        {!isNative && (
```

Comment:
This is confusing (and Ed has mentioned this on Slack) but the when `is_post_native` is true it actually means the post was created on Facebook/Twitter/LinkedIn.

Perhaps to avoid this confusion in Ray we could name this prop `isImported` which seems clearer.